frameit/lib/frameit/options.rb

```diff
@@ -39,7 +39,18 @@ def self.available_options
                            env_name: "FRAMEIT_USE_LEGACY_IPHONE_6_S",
                            is_string: false,
                            description: "Use iPhone 6s frames instead of iPhone 7 frames",
-                           default_value: false)
+                           default_value: false),
+        FastlaneCore::ConfigItem.new(key: :force_orientation_block,
+                           type: Proc,
+                           description: "[Advanced] A block to customize your screnshots' device orientation",
```

Comment:
What does "a block" mean here?

Reply:
In ruby this construct is called a block, usually passed in as a trailing parameter:

```ruby
[1, 2, 3].each do |x|
  puts x
end
```

(the `do ... end` is the actual block)

When you want to pass it as a parameter explicitly you need to wrap it in a `Proc`, though, because you can't assign blocks to variables/parameters:

```ruby
block = proc do |x|
  puts x
end
```